torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8350,6 +8459,20 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     ),
 ]
 
+# Reduction operator database
+reduction_op_db: List[ReductionOpInfo] = [
+    ReductionOpInfo(
```

Comment:
Add this alphabetically (as best you can since the list isn't alphabetical) instead of at the end of the sequence to help avoid/prevent merge conflicts

Reply:
That was my goals with putting it into a reductions_op_db initially, but I guess we should just do a refactor of common_methods_invocations at some point. I'll add them alphabetically.